### PR TITLE
MIAM Tests - First set including all outbound journeys to new section…

### PR DESCRIPTION
**Pull request by JakeCG**
…s, up to the valid reason page

### Change description

MIAM

### JIRA link

[PRL-6371](https://tools.hmcts.net/jira/browse/PRL-6371), [PRL-6372](https://tools.hmcts.net/jira/browse/PRL-6372), [PRL-6373](https://tools.hmcts.net/jira/browse/PRL-6373), [PRL-6374](https://tools.hmcts.net/jira/browse/PRL-6374), [PRL-6375](https://tools.hmcts.net/jira/browse/PRL-6375), [PRL-6376](https://tools.hmcts.net/jira/browse/PRL-6376)

**Before merging a pull request make sure that:**

- [ ] Commits are meaningful and simple
- [ ] All commits are squashed into a single commit
- [ ] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/common/commonStaticText.ts [hunk 1]
```diff
@@ -1,4 +1,10 @@
 export enum CommonStaticText {
+  strippedYes = "Yes",
+  paddedYes = " Yes ",
+  strippedNo = "No",
+  paddedNo = " No ",
+  paddedContinue = " Continue ",
+  paddedSaveAndComeBackLater = " Save and come back later ",
```
Comment (abigailsmith97) [inline e2e/common/commonStaticText.ts:7]: i thought we didn't need 'padded' etc anymore?

File: e2e/journeys/citizen/createCase/C100.ts [hunk 2]
```diff
@@ -64,15 +96,34 @@ export class C100 {
...
       });
       // People
-      await C100Confidentiality.c100Confidentiality({
+      // await C100Confidentiality.c100Confidentiality({
+      //   page: page,
+      //   accessibilityTest: accessibilityTest,
+      //   errorMessaging: errorMessaging,
+      //   c100OthersKnowApplicantsContact: c100OthersKnowApplicantsContact,
+      //   c100PrivateDetails: c100PrivateDetails,
+      // });
```
Comment (abigailsmith97) [inline e2e/journeys/citizen/createCase/C100.ts:105]: is this meant to be commented out?

File: e2e/journeys/citizen/createCase/C100MIAM/MIAM.ts [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+  miamGeneralExemptions: boolean; // Sets the different MIAM exemptions.
+  miamDomesticAbuse: boolean; // Decides whether there are all domestic abuse reasons listed.
+  miamDomesticAbuseProvidingEvidence: boolean;
+  miamChildProtectionConcernsType: MiamChildProtectionConcernsType; // Decides which child protection concern is listed.
+  miamUrgencyType: MiamUrgencyType; // Decides which reason there is for urgency.
+  miamAttendanceType: MiamAttendanceType; // Decides which reason there is for previous MIAM attendance.
+  miamPreviousAttendanceMediatorSignedDocument: boolean; // Doesn't affect journey
+  miamOtherReasonForNotAttending: MiamOtherReasonForNotAttending; // Decides which Other reason there is for not attending
+  miamReasonForNoAccessToMediator: MiamReasonForNoAccessToMediator; // If other reason === "No access to mediator", this decides which reason for no access is used.
+}
```
Comment (abigailsmith97) [inline e2e/journeys/citizen/createCase/C100MIAM/MIAM.ts:58]: comments <3

File: e2e/pages/citizen/createCase/C100/MIAM/miamChildProtectionPage.ts [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukHint}:text-is("${MiamChildProtectionContent.govukHint}")`,
+        1,
+      ),
+    ]);
+    if (accessibilityTest) {
+      await AxeTest.run(page);
+    }
+  }
```
Comment (abigailsmith97) [inline e2e/pages/citizen/createCase/C100/MIAM/miamChildProtectionPage.ts:83]: other pages use
```
if (accessibilityTest) {
      await AccessibilityTestHelper.run(page);
    }
```

File: e2e/pages/citizen/createCase/C100/MIAM/miamChildProtectionPage.ts [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      page: page,
+      miamChildProtectionConcernsType: miamChildProtectionConcernsType,
+    });
+  }
+
+  private static async checkPageLoads({
+    page: page,
+    accessibilityTest: accessibilityTest,
+  }: Partial<MiamChildProtectionPageOptions>): Promise<void> {
+    if (!page) {
```
Comment (abigailsmith97) [inline e2e/pages/citizen/createCase/C100/MIAM/miamChildProtectionPage.ts:50]: is this needed?

**Review by Scralfie — COMMENTED** [review]
Over all it looks okay!

File: e2e/journeys/citizen/createCase/C100.ts (hunk 2 above)
Comment (JakeCG) [reply e2e/journeys/citizen/createCase/C100.ts:105]: Yeah this is needed as the prior pages aren't ready yet

**Review by Scralfie — APPROVED** [review]

**Review by abigailsmith97 — APPROVED** [review]

**Pull request merged**